Football/src/components/ColorSchemeSwitcher/index.tsx

```diff
@@ -1,16 +1,14 @@
 import React from 'react';
-import {Switch, View} from 'react-native';
+import { Switch, View } from 'react-native';
+import { ColorSchemeSwitcherPropsType } from './types';
 
-export const ColorSchemeSwitcher: React.FC = ({
+export const ColorSchemeSwitcher: React.FC<ColorSchemeSwitcherPropsType> = ({
   colorSchemeSwitchAction,
```

Comment:
will be better to use props and destruct params like this:
const { a, b} = props;

Reply:
and further, let's use the same approach